.github/workflows/push-mimir-build-image.yml

```diff
@@ -0,0 +1,103 @@
+name: Build and Push mimir-build-image
+
+# configure trigger by pull request or manually
+on:
+  pull_request:
+    types: [opened, synchronize, ready_for_review]
```

Comment:
I'm curious why does this workflow need run again when PR is marked as "Ready for Review"?

Reply:
I was thinking when someone opened PR and set to ready-for-review later, but in that case it would already have the image up to date. we can remove it.